src/Mindee/MindeeClientOptions.cs

```diff
@@ -54,16 +54,26 @@ public sealed class AsyncPollingOptions
         /// </summary>
         public int MaxRetries { get; }
 
+        /// <summary>
+        /// Wait this many milliseconds between each polling attempt.
+        /// </summary>
+        public int InitialDelayMilliSec { get; }
+
+        /// <summary>
+        /// Wait this many milliseconds before the first polling attempt.
+        /// </summary>
+        public int IntervalMilliSec { get; }
+
         /// <summary>
         /// Options to pass for asynchronous parsing with polling.
         /// </summary>
         /// <param name="initialDelaySec"><see cref="InitialDelaySec"/></param>
         /// <param name="intervalSec"><see cref="IntervalSec"/></param>
         /// <param name="maxRetries"><see cref="MaxRetries"/></param>
-        public AsyncPollingOptions(double initialDelaySec = 4.0, double intervalSec = 2.0, int maxRetries = 30)
+        public AsyncPollingOptions(double initialDelaySec = 4.0, double intervalSec = 1.5, int maxRetries = 60)
```

Comment:
Other defaults are 2s & 60 retries (2 minutes total). I don't mind changing delays on other libs, but I don't really like the timers being different on each

Reply:
Was doing some testing. Good catch, reverted.